app/knight_configure.py

```diff
@@ -0,0 +1,86 @@
+KNIGHTS = {
+    "lancelot": {
+        "name": "Lancelot",
+        "power": 35,
+        "hp": 100,
+        "armour": [],
+        "weapon": {
+            "name": "Metal Sword",
+            "power": 50,
```

Comment:
There is a trailing comma after the 'power' key in the 'weapon' dictionary for 'lancelot'. While Python allows trailing commas, it's generally better to remove them for consistency, especially since other similar entries do not have trailing commas.